Uniswap/interface
# feat: detect more injections

## Description


Adds more injection detection via new `getInjection` utility. This adds:
* Phantom
* Trust Wallet
* Brave Browser
* Ledger Connect
* Rabby

Adds more wallet icon image files, moves them to a new subfolder, and makes them all square. Border radius is now handled by css.


## Screen capture

https://github.com/Uniswap/interface/assets/39385577/1ff60788-14ff-4c88-873e-edb2f7c51119

## Test plan
* Try a variety of injected wallets, confirm the expected icon/name appears
* connect to the app using mobile wallet browsers (but some mobile wallet browsers e.g. brave & phantom cannot be tested properly yet as they will not inject into interface on vercel/local)

### Automated testing
* new unit tests
